tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -227,16 +224,18 @@ def _test_func(
     rank_0_output,
     state_after,
 ):
+    # Make convs more deterministic.
+    torch.backends.cudnn.benchmark = True
+    torch_set_deterministic(True)
+    os.putenv("CUBLAS_WORKSPACE_CONFIG", ":4096:8")
```

Comment:
wow, didn't know about this env variable :)

Reply:
yeah, it was printed out by the pytorch error msg when I don't have it. determinism seems to help avoid some small numerical mismatch on the CI machines (but not locally).